docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,233 @@
+
+### Steps
+
+* Create an Amazon EKS cluster on the basis of [AWS official guide](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html). 
+* Create a managed node group on the basis of [AWS official guide](https://docs.aws.amazon.com/eks/latest/userguide/create-managed-node-group.html) with the above requirements and recommendations.
+* Configure kubectl to connect to your Kubernetes cluster using the `aws eks update-kubeconfig` command.
+    
+   ```console
+    $ aws eks --region <region-code> update-kubeconfig --name <cluster_name>
+   ```
+
+## Step 4. Install Scalar DL
+
+After creating a Kubernetes cluster next step is to deploy Scalar DL into the EKS cluster.
+This section shows how to install Scalar DL to the EKS cluster with [Helm charts](https://github.com/scalar-labs/helm-charts).
+
+### Prerequisites
+
+Install the Helm on your bastion to deploy helm-charts:
+
+* [Helm](https://helm.sh/docs/intro/install/): helm command-line tool to manage releases in the EKS cluster. In this tutorial, it is used to deploy Scalar DL and Schema loading helm charts to the EKS cluster. Helm version 3.5 or latest is required.
+
+### Requirements
+
+* You must have the authority to pull `scalar-ledger` and `scalardl-schema-loader` container images.
+* You must configure the database properties in the helm chart custom values file.
+* You must confirm that the replica count of the ledger and envoy pods in the `scalardl-custom-values.yaml` file is equal to the number of nodes in the Scalar DL node group.
+
+### Steps
+
+The following steps show how to install Scalar DL on EKS:
+
+1. Download the following Scalar DL configuration files from the [scalar-kubernetes repository](https://github.com/scalar-labs/scalar-kubernetes/tree/master/conf). 
+Note that they are going to be versioned in the future, so you might want to change the branch to use a proper version.
+    * scalardl-custom-values.yaml
+    * schema-loading-custom-values.yaml
+ 
+2. Update the database configuration in scalarLedgerConfiguration and schemaLoading sections as specified in [Set up a database guide](./SetupAWSDatabase.md#Configure-Scalar-DL).
+3. Create the docker-registry secrets for pulling the Scalar DL images from the GitHub registry
+    
+   ```console
+    $ kubectl create secret docker-registry reg-docker-secrets --docker-server=ghcr.io --docker-username=<github-username> --docker-password=<github-personal-access-token>
+   ```
+
+4. Run the Helm commands on the bastion machine to install Scalar DL on EKS
+    
+   ```console
+    # Add Helm charts
+    $ helm repo add scalar-labs https://scalar-labs.github.io/helm-charts
+    
+    # List the Scalar charts.
+    $ helm search repo scalar-labs
+    
+    # Load Schema for Scalar DL install with a release name `load-schema`
+    $ helm upgrade --install load-schema scalar-labs/schema-loading --namespace default -f schema-loading-custom-values.yaml
```

Comment:
Maybe better to add `--version` as an option or use `helm install` command for prevent to upgrade latest.

Reply:
Thank you added `--version` option with the command in b1fc1b7